dask_cuda/benchmarks/local_cudf_shuffle.py

```diff
@@ -0,0 +1,243 @@
+from collections import defaultdict
```

Comment:
As a general note for this file since it overlaps a lot with https://github.com/rapidsai/dask-cuda/blob/branch-0.18/dask_cuda/benchmarks/local_cudf_merge.py, would it make sense to combine both? I did something similar for the CuPy benchmarks adding a new `--operation` option in https://github.com/rapidsai/dask-cuda/blob/7d27a0c320dcf093d0c51089c09223c17424d52a/dask_cuda/benchmarks/local_cupy.py#L211-L217 .

Reply:
I thought about it but while profiling and experimenting I think it is best to have them as two individual files. 
Furthermore, I suspect that in the future merge will be using shuffle, which will make the merge benchmark less important.

Reply:
> I thought about it but while profiling and experimenting I think it is best to have them as two individual files.

Why? I think it will really make our lives easier to reduce code duplication.

Reply:
Sorry for not being clear. The thing is, the current version of the merge benchmark is fairly complex and I would like to simplify it before combining the two benchmarks. I am currently working on a redesign of explicit-comms that will make shuffle compatible with regular Dask and make it possible to write the merge benchmark as a regular Dask merge.

In short, I totally agree with your sentiment I just want to delay the unification a bit :)

Reply:
Got it, thanks for the details. At one point, I think we could even make the whole benchmark unified, creating cluster, etc., is something that's pretty much the same for all benchmarks, eventually I want to work on making that happen.